HW4/array_manipulations.cpp

```diff
@@ -0,0 +1,94 @@
+#include "array_manupulations.h"
+#include <chrono>
+#include <iomanip>
+#include <iostream>
+#include <random>
+
+void output_array_range(unsigned int *begin, unsigned int *end) {
+  if (begin == nullptr || end == nullptr)
+    return;
+  for (auto it = begin; it != end; ++it)
```

Comment:
```suggestion
  for (auto it {begin}; it != end; ++it)
```
Після стандарту C++11 додано універсальну ініціалізацію

Reply:
Верно, спасибо!